app/models/student_loans/eligibility.rb

```diff
@@ -71,7 +71,9 @@ def ineligible?
         not_taught_enough?
     end
 
-    def ineligibility_reason
+    def ineligibility_reason(tried_all_schools = nil)
```

Comment:
I feel like this change subverts this model method and couples it a bit more to the journey than before, which might not be that helpful. Rather than modifying this method on the model, how about if we just have an `if` in the ineligibility template, something like:

```ruby
if params[:tried_all_schools]
  # render the generic reason
else
  # render the specific one
end
```

That keeps the nitty gritty of the evolving journey out of this method, which to know has only been about telling us why the thing is ineligible without coupling it to the thing the user sees in the UI.